pkgs/tools/games/ajour/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, autoPatchelfHook
+, cmake
+, pkg-config
+, expat
+, openssl
+, libxkbcommon
+, vulkan-headers
+, vulkan-loader
+, wayland
+, x11
+, xorg
+, python3
+}:
+
+let
+  rpathLibs = [
+    libxkbcommon
+    vulkan-loader
+    wayland
+  ];
+
+in rustPlatform.buildRustPackage rec {
+  pname = "Ajour";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "casperstorm";
+    repo = "ajour";
+    rev = version;
+    sha256 = "14zh5m4knghvmrgbkmfc3mq4sbhkirzsb1pqycfcdlijf8rq7qjr";
+  };
+
+  cargoSha256 = "0njly8iv71gqzipls4hnvzvn6p58w0figww5s8ya76xbbqpi7vjm";
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    cmake
+    pkg-config
+    python3
+  ];
+
+  buildInputs = [
+    expat
+    openssl
+    xorg.libxcb
+    x11
+  ];
+
+  fixupPhase = ''
+    patchelf --set-rpath "${lib.makeLibraryPath rpathLibs}:$(patchelf --print-rpath $out/bin/ajour)" $out/bin/ajour
+  '';
+
+  meta = with lib; {
+    description = "World of Warcraft addon manager written in Rust";
+    longDescription = ''
+      Ajour is a World of Warcraft addon manager written in Rust with a
+      strong focus on performance and simplicity. The project is
+      completely advertisement free, privacy respecting and open source.
+    '';
+    homepage = "https://github.com/casperstorm/ajour";
+    changelog = "https://github.com/casperstorm/ajour/blob/master/CHANGELOG.md";
+    licenses = license.mit;
```

Comment:
```suggestion
    license = license.mit;
```

Reply:
Nice catch! It also looks like `license.mit` should be changed to `licenses.mit`. It seems like the words were just swapped.